.gitlab-ci.yml

```diff
@@ -30,6 +30,7 @@ build:
     - output/seed-windows-amd64
   script:
     - ./build-cli.sh $CI_COMMIT_TAG
+    - ./generate-manpage.sh
```

Comment:
Can we just use the same approach as the travis build? We should allow the image to be overridden by using an environment variable.